### PR TITLE
pr: add page

### PR DESCRIPTION
**Pull request by monst12**
New page for command `pr` of GNU Core Utilities as listed in #2213.
I would be very happy about feedback.

- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pr.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# pr
+
+> Paginate or columnate files for printing.
+> More information: <https://www.gnu.org/software/coreutils/manual/html_node/pr-invocation.html>.
+
+- Print multiple files with a default header and footer:
+
+`pr {{file1 file2 file3}}`
+
+- Print all files in parallel, one in each column without a header or footer:
```
Comment (agnivade) [inline pages/common/pr.md]: parallel is technically the wrong word here. Perhaps better may be "Print all files together, .."

File: pages/common/pr.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`pr -m -T {{file1 file2 file3}}`
+
+- Print all files beginning at page 2 up to page 5 with a given page length (including header and footer):
+
+`pr +{{2}}:{{5}} -l {{page_length}} {{file1 file2 file3}}`
+
+- Print all files with an offset each line and a truncating custom page width:
```
Comment (agnivade) [inline pages/common/pr.md]: with an offset _for_ each line ..

And we can get rid of "all files". It is implicit after being mentioned 2 times above.

File: pages/common/pr.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`pr -o {{offset}} -W {{width}} {{file1 file2 file3}}`
+
+- Print files with numbered lines, a custom centered header and date format:
```
Comment (agnivade) [inline pages/common/pr.md]: I think we can split this into 2 separate examples - one for custom header, one for date format. And same applies as above: we can get rid of "files"

File: pages/common/pr.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+# pr
+
+> Paginate or columnate files for printing.
+> More information: <https://www.gnu.org/software/coreutils/manual/html_node/pr-invocation.html>.
+
+- Print multiple files with a default header and footer:
+
+`pr {{file1 file2 file3}}`
```
Comment (sbrl) [inline pages/common/pr.md]: I'd be tempted to use `{{file1}} {{file2}} {{file3}}` instead of `{{file1 file2 file3}}` here.

Comment (schneiderl) [reply pages/common/pr.md]: Reads better and I think that's how we deal with this in the majority of the pages.

Comment (monst12) [reply pages/common/pr.md]: I used the tar page as reference for mutliple files input, but I agree that it would look nicer with separated tokens.

Comment (sbrl) [reply pages/common/pr.md]: Ah, interesting @monst12! Perhaps we should update the `tar` page then.

**Comment by sbrl** [comment]
Are we good to merge, @monst12?

File: pages/common/pr.md (hunk 4 above)
Comment (sbrl) [reply pages/common/pr.md]: Sure, go ahead! A PR would be great @monst12 :D

**Comment by sbrl** [comment]
Merging, as @agnivade's comments appear to have been addressed, wee have a high volume of PRs at the moment, and this one has been sitting around for a while.

**Pull request merged**